python/popgetter/metadata.py

```diff
@@ -10,6 +11,64 @@
 from pydantic import BaseModel, Field, computed_field, model_validator
 
 
+class COL(Enum):
+    """
+    This class stores the eventual column names of all the metadata classes,
+    which are used when serialising the metadata to a dataframe.
+    """
+
+    GEO_ID = "GEO_ID"
+
+    COUNTRY_ID = "country_id"
+    COUNTRY_NAME_SHORT_EN = "country_name_short_en"
+    COUNTRY_NAME_OFFICIAL = "country_name_official"
+    COUNTRY_ISO3 = "country_iso3"
+    COUNTRY_ISO2 = "country_iso2"
+    COUNTRY_ISO3166_2 = "country_iso3166_2"
+
+    PUBLISHER_ID = "data_publisher_id"
+    PUBLISHER_NAME = "data_publisher_name"
+    PUBLISHER_URL = "data_publisher_url"
+    PUBLISHER_DESCRIPTION = "data_publisher_description"
+    PUBLISHER_COUNTRIES_OF_INTEREST = "data_publisher_countries_of_interest"
+
+    GEOMETRY_ID = "geometry_id"
+    GEOMETRY_FILENAME_STEM = "geometry_filename_stem"
```

Comment:
Since we will rerun the pipeline, perhaps we should include #129 here too and revise this to:
```suggestion
    GEOMETRY_FILEPATH_STEM = "geometry_filepath_stem"
```
(along with the associated changes elsewhere)

Reply:
Yes, happy to! I'Il push a new commit.

Reply:
Thanks!